media_kit/lib/src/models/player_state.dart

```diff
@@ -64,6 +64,12 @@ class PlayerState {
   /// Currently available video, audio & subtitle tracks.
   final Tracks tracks;
 
+  /// Currently playing video's width.
+  final int? width;
```

Comment:
I added two new event streams `width` & `height` for retrieving resolution of the video.

This is used within package:media_kit_video for Android, since there is no render loop on native side unlike Windows; GNU/Linux, macOS or iOS.

Also: A new feature!